### ISSUE TITLE
History of old assets

### ISSUE THREAD

**Issue by CyberPoly1000** [intuitem/ciso-assistant-community#295]
**Problem statement**
I would like to be able to have a traceability of old assets. For some audits, it is required to see what was the previous status of the company and how it has evolved until now. When deleting an asset, it is currently totally erased. An option to be able to keep it in an history bucket could be great. 

**Expected behavior**

1. Delete an asset
2. Pop-up appears with an option to move the asset to the history dump or not

**Comment by eric-intuitem** [comment]
Indeed, it shall not be possible to delete an asset that is used in a risk analysis. But it could have an "obsolete" flag to make it impossible to use in any new analysis.
